LICENSE.md

```diff
@@ -1,7 +1,7 @@
-The "borales/yii2-phone-input" is free software.
+The "aklim/yii2-phone-input" is free software.
 It is released under the terms of the following BSD License.
 
-Copyright © 2014 by Alexandr Bordun. All rights reserved.
+Copyright © 2017 by Andrey Klimenko. All rights reserved.
```

Comment:
@aklim I don't think stealing someone's code and changing the copyright is a good idea